internal/repository/user/human_email.go

```diff
@@ -145,8 +147,19 @@ func NewHumanEmailCodeAddedEvent(
 	aggregate *eventstore.Aggregate,
 	code *crypto.CryptoValue,
 	expiry time.Duration,
+	authRequestID string,
 ) *HumanEmailCodeAddedEvent {
-	return NewHumanEmailCodeAddedEventV2(ctx, aggregate, code, expiry, "", false)
+	return &HumanEmailCodeAddedEvent{
```

Comment:
Why is the V2 event not used anymore here?

Reply:
i had to split it up, since v2 events never have a authRequestID and only v2 will ever have urlTemplate and codeReturn